source/RialtoGStreamerMSEAudioSink.cpp

```diff
@@ -524,7 +575,7 @@ static void rialto_mse_audio_sink_set_property(GObject *object, guint propId, co
     case PROP_SYNC_OFF:
     {
         priv->syncOff = g_value_get_boolean(value);
-        if (!client)
+        if (!client || !basePriv->m_sourceAttached)
```

Comment:
why source needs to be attached, wouldn't caching on the RialtoServer side be enough?